src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
-
+import Header from './Components/Header';
+import Landing from './Components/Landing';
+import ShowProducts from './Components/ShowProducts';
+import Cart from './Components/Cart';
+import { Route, Routes } from 'react-router-dom';
+import CartProvider from './Context/CartProvide';
 function App() {
```

Comment:
please add space here, and make sure to format each the file before pushing them